back/engines/commercial/user_custom_fields/app/controllers/user_custom_fields/web_api/v1/ref_distributions_controller.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module UserCustomFields
+  module WebApi::V1
+    class RefDistributionsController < ::ApplicationController
+      def show
+        render json: serialize(reference_distribution)
+      end
+
+      def create
+        ref_distribution = authorize(Representativeness::RefDistribution.new(create_params))
+
+        Representativeness::RefDistribution.transaction do
+          remove_reference_distribution_if_any(params[:custom_field_id])
+          ref_distribution.save!
+        end
+
+        side_fx.after_create(ref_distribution, current_user)
+        render json: serialize(ref_distribution), status: :created
+      rescue ActiveRecord::RecordInvalid => e
+        render json: { errors: e.record.errors.full_messages }, status: :unprocessable_entity
+      end
+
+      def destroy
+        reference_distribution.destroy!
+        side_fx.after_destroy(reference_distribution, current_user)
```

Comment:
`before_destroy` is missing from this method.

Reply:
This is similar to `before_create`. `before_destroy` does not exist on the side fx class.